### PR TITLE
Improve readability of documentation

### PR DESCRIPTION
**Pull request by CollieIsCute**
According to [this page of unr tutorial](https://www.unr.edu/writing-speaking-center/writing-speaking-resources/parenthetical-phrases), parenthetical phrases in the middle of sentence is good to go with commas before and after themselves.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ TIME     EVENT COMM             PID     PPID    FILENAME/EXIT CODE
 ## uprobe
 
 `uprobe` is an example of dealing with user-space entry and exit (return) probes,
-`uprobe` and `uretprobe` in libbpf lingo. It attached `uprobe` and `uretprobe`
+`uprobe` and `uretprobe`, in libbpf lingo. It attached `uprobe` and `uretprobe`
```
Comment (anakryiko) [inline README.md:114]: let's also fix attached -> attaches typo, while at it?

Also, for Signed-off-by in commit message, you are supposed to use properly capitalized real name, if that's not a problem. Thanks.

**Comment by anakryiko** [comment]
@CollieIsCute do you intend to send an updated version with valid Signed-off-by tag?

**Comment by CollieIsCute** [comment]
Sorry for a late reply. I am taking a vacation rn. I'll fix this PR before 12/10

File: README.md (hunk 1 above)
Comment (CollieIsCute) [reply README.md]: fixed.

**Comment by CollieIsCute** [comment]
I've fix all the issues, pls have a look : )

**Comment by CollieIsCute** [comment]
wait a second. there are still some cicd test issues to be fix

**Comment by CollieIsCute** [comment]
ok, I think I just resolved all of them.

**Pull request merged**